### ISSUE TITLE
Tiled map tutorial: Import object types is different in 1.9.2

### ISSUE THREAD

**Issue by Ethanol6** [flareteam/flare-engine#1845]
I'm following along with the tutorial to create maps with Tiled. However, there is no longer a menu item under "view" for "import object types". I found no way of importing the `objecttypes.xml` file.

Also, it would appear that the `type` attribute has been replaced by `class`.
https://doc.mapeditor.org/en/stable/manual/custom-properties/#custom-classes

**Comment by dorkster** [comment]
It looks like it got renamed, but the feature is still there. Go to `View -> Custom Types Editor` and then click `Import`. In the file dialog, you'll need to change the filter from `Custom Types files` to `Object Types XML`. I added a note to the wiki page to reflect this.

Didn't know about the change from "type" to "class". The old format should still work, at least with Tiled 1.9.2.

**Comment by Ethanol6** [comment]
Yeah, I see that option but it is greyed out. Doesn't seem to change depending on what layers or objects I have selected.

**Comment by dorkster** [comment]
I see. It looks like that feature only works if you're working in the context of a project. So you'll need to create project with `File -> New -> New Project` first, and then add the folder which contains your map to it.
